2018-10-02  Gerald Pfeifer  <gerald@pfeifer.com>

	* io/close.c [!HAVE_UNLINK_OPEN_FILE]: Include <string.h>.

--- UTC
Index: libgfortran/io/close.c
===================================================================
--- libgfortran/io/close.c	(revision 264799)
+++ libgfortran/io/close.c	(revision 264800)
@@ -26,6 +26,9 @@ see the files COPYING3 and COPYING.RUNTIME respect
 #include "unix.h"
 #include "async.h"
 #include <limits.h>
+#if !HAVE_UNLINK_OPEN_FILE
+#include <string.h>
+#endif
 
 typedef enum
 { CLOSE_DELETE, CLOSE_KEEP, CLOSE_UNSPECIFIED }
